### ISSUE TITLE
Stopped working for Fortnite after new update

### ISSUE THREAD

**Issue by shinenami** [lociii/homeassistant-overwolf-status#5]
Says game listener started but does not send nor show any events besides
Error sending events: TypeError: Failed to fetch

**Comment by lociii** [comment]
This is more of an issue with https://github.com/lociii/overwolf-homeassistant-webhook as this repo is just the custom component that receives the webhooks and fires events in HomeAssistant.

**Comment by lociii** [comment]
Are you sure that your webhook URL is set properly? This messages is shown when a game event is received and should be forwarded to HomeAssistant but the network communication fails.

**Comment by lociii** [comment]
See https://github.com/lociii/overwolf-homeassistant-webhook/blob/main/src/services/publisher.ts#L76

**Comment by shinenami** [comment]
> This is more of an issue with https://github.com/lociii/overwolf-homeassistant-webhook as this repo is just the custom component that receives the webhooks and fires events in HomeAssistant.

Sorry, its just the only repo i found in HA Game Events description in overwolf.

Actually, it doesn't show the "Error sending events: TypeError: Failed to fetch"(I use local webhook URL and didn't start it last time).
But yeah no events are showing even in HAGE event log itself, but it was working before the recent FN update and I didn't change anything.

Previously there were many events going on in event log, even when you're just running it would log coordinates, but now it only says: 
> APP: Game event listener started for title: Fortnite Battle Royale
> INFO: {"gep_internal":{"version_info":{"local_version":"258.1.1","public_version":"258.1.1","is_updated":true}}}
> APP: Game event listener stopped: {"gameInfo":... (game and account info when i exit game)}

Not sure what happened and how to debug it

**Comment by shinenami** [comment]
I found out this is not problem on HAGE side but with overwolf itself. [All events are currently unavailable ](https://overwolf.github.io/status/fortnite)

**Comment by lociii** [comment]
@shinenami most events should be supported by Overwolf again. Can you please check and confirm?  
I eventually have to scrape events again.

**Comment by shinenami** [comment]
> @shinenami most events should be supported by Overwolf again. Can you please check and confirm? I eventually have to scrape events again.

Still not working for me

**Comment by lociii** [comment]
I just scraped the game features again and cannot find any issue with the app itself. Can you please test for other games?

**Comment by shinenami** [comment]
Okay i just tested in Lethal Company (it was working), and weirdly, after that it started working in Fortnite too.
But there was a different problem, the only event working was "message_feed"
Then I relaunched Fortnite to see if other events would start working and that time even "message_feed" wasn't working
Then I tested it in Lethal Company again, and it was working there. 
So I launched Fortnite once again and now it works for all events except "hit". Weird.

Tested it a few more times and all events are working except "hit"
